collection/snitch.yml

```diff
@@ -27,6 +27,13 @@
       tags:
         - config
 
+- hosts: all
+  tasks:
+    - name: Collect values from interface configurations
+      action: configuredinterface_snitch
```

Comment:
I think 'action:' is not needed here. Action indicates an action plugin that starts on where ever ansible is running.  Maybe just
```yaml
- hosts: all
  tasks:
    - name: Collect values from interface configurations
      configuredinterface_snitch:
      tags:
        - config
```